staging/src/k8s.io/kube-aggregator/pkg/apiserver/apiserver.go

```diff
@@ -20,6 +20,8 @@ import (
 	"net/http"
 	"time"
 
+	openapicommon "k8s.io/kube-openapi/pkg/common"
```

Comment:
nit: shouldn't be added at the end of the import statement?

Reply:
it's an external library.